whos-online/whos-online.js

```diff
@@ -0,0 +1,51 @@
+const whosOnline = (friends) => {
+  
+  friends = [
+     {
+      username: 'Lucy',
+      status: 'offline',
+      lastActivity: 22
+    }, {
+      username: 'Bob',
+      status: 'online',
+      lastActivity: 104
+    }
+  ]
+
+  function userNewStatus(item){
+    if (item.lastActivity > 10 && item.status != 'offline') {
+      item.status = 'away';
+    };
+  }
+  friends.map(userNewStatus);
+
+  function getUsername(item) {
```

Comment:
Same here, this function doesn't depends of any variable that exist in this scope, so you could move it outside `whosOnline`.

Reply:
Same as the above comment, would it get executed in CW? If so, I should move it before `whosOnline`, right?

Reply:
Right.